src/test/java/org/prebid/server/bidder/colossus/ColossusBidderTest.java

```diff
@@ -149,10 +152,12 @@ public void makeBidsShouldReturnEmptyListIfBidResponseSeatBidIsNull() throws Jso
     }
 
     @Test
-    public void makeBidsShouldReturnBannerBidByDefault() throws JsonProcessingException {
+    public void makeBidsShouldReturnBannerBid() throws JsonProcessingException {
         // given
         final HttpCall<BidRequest> httpCall = givenHttpCall(
-                BidRequest.builder().imp(singletonList(Imp.builder().id("123").build())).build(),
+                BidRequest.builder().imp(singletonList(Imp.builder().id("123")
```

Comment:
`BidRequest.builder().imp(singletonList(givenImp(identity()))).build()` change into something like this. Check all other cases, where it is possible to do so.